schemas/fingerprint-server-api.yaml

```diff
@@ -954,18 +954,6 @@ components:
         - asn
         - network
       title: ASN
-    DataCenterInfo:
-      type: object
-      additionalProperties: false
-      properties:
-        result:
-          type: boolean
-        name:
-          type: string
-          example: 'DediPath'
-      required:
-        - result
-      title: DataCenterInfo
     DataCenter:
       type: object
       deprecated: true
```

Comment:
We still need "description: '`dataCenter` is deprecated in favor of `datacenter`", just because the current SDKs use `dataCenter` correct?

Reply:
Nice catch! Just missed it =(

Reply:
nice, thank you!